terraform.tfvars.example

```diff
@@ -0,0 +1,7 @@
+env=""
+project_id=""
+gcp_billing_account="010C5C-D2E6F2-FA1F92"
+gcp_folder_id="979069919925"
+stackdriver_workspace=""
```

Comment:
Actually, this should be removed and we should be using `project_id` because currently, you will not be able to reference `log-based metrics` from a different project.